src/modules/functions.js

```diff
@@ -0,0 +1,96 @@
+import Task from './tasks.js';
+
+let todolist = [];
+const descriptioninput = document.getElementById('task');
+
+function addtodolist() {
+  const index = todolist.length + 1;
+  const completed = false;
+  const newtask = new Task(completed, descriptioninput.value, index);
+  todolist.push(newtask);
+  localStorage.setItem('newtask', JSON.stringify(todolist));
+  descriptioninput.value = '';
+}
```

Comment:
Nice job @edgardsoza 😉 , but for this function, it's will be better if you use the arrow function